content/vsphere-cpi.md

```diff
@@ -189,6 +190,7 @@ Schema:
     * **password** [String, required]: The login password for the NSX server.
     * **ca_cert** [String, optional]: A CA certificate that can authenticate the NSX server certificate. **Required** if the NSX Manager has a self-signed SSL certificate. Must be in PEM format.
 * **enable\_auto\_anti\_affinity\_drs\_rules** [Boolean, optional]: Creates DRS rule to place VMs on separate hosts. DRS Automation Level must be set to "Fully Automated"; does not work when DRS is set to "Partially Automated" or "Manual". May cause VMs to fail to power on if there are more VMs than hosts after initial deployment. Default: `false`. Available in v33+.
+* **upgrade\_hw\_version** [Boolean, optional]: Upgrades the virtual hardware version of a virtual machine to the latest supported version on the ESXi host. This property can also be specified in VM Types which will override the global configuration. By default this property is false.
```

Comment:
This should be just:
 Upgrades the hardware version of a virtual machine to the latest supported version on the ESXi host. Default: `false`

Reply:
What about override information?

Reply:
Override information is provided in cloud config section. The usual syntax is to describe what the property is here and if its being overridden by cloud config, you specify there which property it is overriding